content/blogs/2024-10-DD-workflow-tools-comparison.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Best 3 Workflow Orchestration Tools: a Comparison"

```

Comment:
I'd word this specifically as data orchestration

```suggestion
title: "Data Orchestrators: How to Choose"
```

Reply:
Same for the rest, reword this to be more focused on data

Reply:
Use this to help: https://kestra.io/use-cases/data-engineers

Reply:
this is an interesting point -- I think we could keep this article focused on workflow orchestration and have a separate one about data orchestrators;  in that case, Federico doesn't need to talk about Dagster here since the focus would be on orchestrating workflows, not data - WDYT @wrussell1999?

Reply:
Agreed - let’s do this as a workflow orchestration one first and do data as a separate PR that includes Dagster. Cc @federico-trotta

Reply:
![use-cases-data-engineers](https://github.com/user-attachments/assets/16b30c47-abcc-49f4-87a3-c032ef1243e7)
Just for the record: the page is broken... :(

Reply:
Got an issue open to fix it, thanks